src/bci_build/package/__init__.py

```diff
@@ -179,6 +179,12 @@ def lifecycle_data_pkg(self) -> list[str]:
             return ["lifecycle-data-sle-module-development-tools"]
         return []
 
+    @property
+    def common_devel_packages(self) -> list[str]:
+        """Returns a list of common development packages that are needed for
+        all development containers"""
+        return ["findutils", "gawk", "git-core", "curl", "procps"]
```

Comment:
I think this should be a constant and not a property of `OsVersion` as it is completely version agnostic

Reply:
it right now is, but it might change, package names could be different between major versions, and we have already other similar functions here on os_version. Are you saying it should be a constant until it has a version dependency even though all other similar functions are in os_version ?